pkg/reconcile/reconcile.go

```diff
@@ -21,6 +21,7 @@ import (
 	"time"
 
 	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/logical-cluster"
```

Comment:
Where is the code for this btw?

Reply:
I think here: https://github.com/vincepri/logical-cluster/tree/define-logical-cluster (I assume just for now as long as it's all WIP)